scripts/test-ci

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+# -p flag stands for --project-name and will create a different network for the test container
+# so it won't interfere with the app container
+docker-compose -p gp-finder-ci-test -f docker-compose-tests.yml down -v
+docker-compose pull gp-finder-db
+docker-compose -p gp-finder-ci-test -f docker-compose-tests.yml up -d gp-finder-db
+docker-compose -p gp-finder-ci-test -f docker-compose-tests.yml build ci-tests
```

Comment:
If the comment about renaming `ci-tests` to `test-ci` is heeded this will need changing. And, just for consistency I'd suggest updating the project name flag to `gp-finder-test-ci` or maybe just `test-ci` - for the sake of consistency.